### PR TITLE
speed up tests by 20 seconds

### PR DESCRIPTION
**Pull request by dannyroberts**
by settings `SOUTH_TESTS_MIGRATE = False`

@czue do you remember why we didn't have this set?

Before this the overhead of just running a single db test is 30+ seconds of setup; with this it's down to ~11.

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -941,9 +939,9 @@
     "mvp-potou": ['mvp_indicators'],
 }
 
-####### South Settings #######
-#SKIP_SOUTH_TESTS=True
-#SOUTH_TESTS_MIGRATE=False
+####### South Settings ###### #
```
Comment (dimagimon) [inline settings.py]: E265 block comment should start with '# '

**Comment by czue** [comment]
I think the main reason is that it prevented us from detecting bugs in our migrations. I think we turned it back on after that happened once. Seems like the 20 seconds is worth it on travis and we should just encourage devs to turn it off locally.

File: settings.py [hunk 2]
```diff
@@ -891,6 +889,11 @@
 # override for production
 DEFAULT_PROTOCOL = 'http'
 
+####### South Settings #######
```
Comment (dimagimon) [inline settings.py]: E265 block comment should start with '# '

**Comment by dannyroberts** [comment]
Ok, so I moved `SOUTH_TESTS_MIGRATE = False` to above the localsettings import, and added `SOUTH_TESTS_MIGRATE = True` to .travis/localsettings.py. I think that's basically what you suggested, but with less hassle remembering to tell it to new devs, etc. People can still run revert to using the more conservative behavior by adding `SOUTH_TESTS_MIGRATE = True` to their localsettings. Sound good?

**Comment by benrudolph** [comment]
+1 to using SOUTH_TESTS_MIGRATE. saved me loads of time on tests

**Comment by czue** [comment]
yep this looks perfect. i'll wait for build just for kicks.

**Pull request merged**